packages/ui/src/components/va-affix/VaAffix.vue

```diff
@@ -34,16 +35,18 @@ export default defineComponent({
   name: 'VaAffix',
   emits: ['change'],
   props: {
-    offsetTop: { type: Number as PropType<number>, default: undefined },
-    offsetBottom: { type: Number as PropType<number>, default: undefined },
+    offsetTop: { type: Number, default: undefined },
+    offsetBottom: { type: Number, default: undefined },
     target: { type: [Object, Function] as PropType<Target | (() => Target)>, default: getWindow },
   },
   setup (props, { emit }) {
+    const element = shallowRef<HTMLElement>()
+
     const getTargetElement = () => (typeof props.target === 'function' ? props.target() : props.target)
 
     const isAffixed = computed(() => state.value.isTopAffixed || state.value.isBottomAffixed)
 
-    const state: Ref<State> = ref({
+    const state = ref<State>({
```

Comment:
Not reactive?

Reply:
Deeply within the work of this task, I do not refactor the functional, because there are no obvious problems in the operation of the component